### PR TITLE
fix(backend): removed settings hacks

### PR DESCRIPTION
**Pull request by ak239**
We used to create custom user data dir for each run, so we can run
more then one ndb window at the same time. We can achieve the same
result by reusing the same app with multiple windows.

- removed robust user data dir handling,
- cleanup settings related code,
- added couple todos for followup,
- fixed proper title for each ndb process for debugging.

### CODE REVIEW

File: lib/launcher.js [hunk 1]
```diff
@@ -4,120 +4,78 @@
...
+    app = await carlo.launch({
+      bgcolor: '#242424',
+      channel: ['stable', 'canary', 'chromium'],
+      title: JSON.stringify({
+        cwd: process.cwd(),
+        argv: process.argv
+      })
+    });
   } catch (e) {
+    console.log(e.message);
```
Comment (pavelfeldman) [inline lib/launcher.js]: Only print it in case message is unexpected (Blha blah already running)

File: lib/launcher.js [hunk 2]
```diff
@@ -4,120 +4,78 @@
...
-      try {
-        removeFolder.sync(userDataDir);
-        break;
-      } catch (e) {
-      }
-    }
+  app.on('window', async window => {
+    const backend = new Backend(window);
+    backends.add(backend);
+    const info = await window.evaluate(() => document.title);
```
Comment (pavelfeldman) [inline lib/launcher.js]: I'd only pass front_end params via url, the rest shoud go through backend.

Comment (ak239) [reply lib/launcher.js]: done!

File: lib/launcher.js (hunk 1 above)
Comment (ak239) [reply lib/launcher.js]: done

**Pull request merged**